cmd/dep/godep_importer.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/pkg/errors"
 )
 
-const godepJSONName = "Godeps.json"
+const godepPath = "Godeps/Godeps.json"
```

Comment:
IMO it's fine to have this be a var instead of a const - we know not to misuse it. so:

```go
var godepPath = filepath.Join("Godeps", "Godeps.json")
```